.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: csharp
```

Comment:
It seems you need to [add the build.sh file](https://rimdev.io/building-net-core-on-travis-ci/#buildsh).